iqmotion/clients/client_files/hobby_input.json

```diff
@@ -1,6 +1,10 @@
 [
 
-{"type_idn":76, "param":"allowed_protocols",	"param_idn":  0, "format":"B", "unit": ""  },
-{"type_idn":76, "param":"protocol",	          "param_idn":  1, "format":"B", "unit": ""  }
+  {"type_idn":76, "param":"allowed_protocols",	      "param_idn":  0, "format":"B", "unit": ""    },
```

Comment:
See my comments on this PR for details on the units that each of these should have: https://github.com/iq-motion-control/iq-module-communication-doc/pulls